numpydoc/numpydoc.py

```diff
@@ -184,6 +183,33 @@ class NumpyCDomain(ManglingDomainBase, CDomain):
     }
 
 
+def match_items(lines, content_old):
+    """Create the right items for lines.
```

Comment:
will do. These are really "Best guess".
So the explanation will be "most of the lines are the same after mangling, but we delete and insert some blank lines. This function tries to match up the blank lines. Any other changes, like including figures or reformatting references will lead to wrong line numbers"